docs/en/guides/pipeline-integrations/administration/file-system-config-reference.md

```diff
@@ -105,29 +105,23 @@ Let's dive deeper into these modes.
 
 For a dynamic folder which corresponds to a {% include product %} query, use the following syntax in your YAML file:
 
-    <a name="the type of dynamic content"></a>
+```YAML
     # the type of dynamic content
     type: shotgun_entity
     
-    <a name="the {% include product %} entity type to connect to"></a>
     # the {% include product %} entity type to connect to
     entity_type: Asset
     
-    <a name="the {% include product %} field to use for the folder name"></a>
     # the {% include product %} field to use for the folder name
     name: code
     
-    <a name="{% include product %} filters to apply when getting the list of items"></a>
     # {% include product %} filters to apply when getting the list of items
-    <a name="this should be a list of dicts, each dict containing"></a>
     # this should be a list of dicts, each dict containing
-    <a name="three fields: path, relation and values"></a>
     # three fields: path, relation and values
-    <a name="(this is std {% include product %} API syntax)"></a>
     # (this is std {% include product %} API syntax)
-    <a name="any values starting with $ are resolved into path objects"></a>
     # any values starting with $ are resolved into path objects
     filters: [ { "path": "project", "relation": "is", "values": [ "$project" ] } ] 
+```
```

Comment:
Watching the preview, this snippet is better rendered when removing the quotes and the YAML label, like the snippet starting at line 140.

Reply:
![image](https://user-images.githubusercontent.com/2762494/156184129-58080f68-66d6-4134-afb8-378270cd0295.png)

![image](https://user-images.githubusercontent.com/2762494/156184179-a6041dfa-a65a-43d7-87b2-026f203b1fa4.png)